pyglet/shapes.py

```diff
@@ -431,16 +449,18 @@ def batch(self, batch):
 class Arc(ShapeBase):
     _draw_mode = GL_LINES
 
-    def __init__(self, x, y, radius, segments=None, angle=math.tau, start_angle=0,
-                 closed=False, color=(255, 255, 255, 255), batch=None, group=None):
+    def __init__(self, x: number, y: number, radius: float, segments: Optional[int]=None, 
+                 angle: float=math.tau, start_angle: float=0, closed: bool=False, 
+                 color: Tuple[int, int, int, int]=(255, 255, 255, 255), 
+                 batch: Optional[Batch]=None, group: Optional[Group]=None) -> None:
```

Comment:
add gap between type hint and default value

Reply:
I think that will be a waste of time. It will make the lines longer for no reason.